thrift.sh

```
@@ -3,7 +3,7 @@ version: "%(tag_basename)s"
 source: https://git-wip-us.apache.org/repos/asf/thrift.git
 tag: 0.9.3
 build_requires:
- - "GCC-Toolchain:(?!osx)"
+ - GCC-Toolchain
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)